tests/run_tests/driver_dc_test.py

```diff
@@ -53,7 +53,7 @@ def test_dc_fwr_run(
         geoh5=geoh5,
         mesh=model.parent.uid,
         topography_object=topography.uid,
-        z_from_topo=True,
+        z_from_topo=False,
         data_object=survey.uid,
         starting_model=model.uid,
         resolution=None,
```

Comment:
need to send `drape_height=0`, otherwise we are getting air cells

Reply:
done